### ISSUE TITLE
Custom Theming support

### ISSUE THREAD

**Issue by Noor0** [a85/POSTMan-Chrome-Extension#4318]
it'd be great to be able to customize postman colors through a json config file or through setting. Current dark theme is just ugly

**Comment by prashantagarwal** [comment]
@Noor0 Thanks for suggesting this feature. This is definitely on our roadmap.

**Comment by sankalp0o** [comment]
Hi @Noor0, we recently refactored the colors in both the themes which hopefully made the dark theme a little better. This also unblocked us from being able to manipulate the theme by adjusting colors in a single place and we'll soon be able to expose this which will allow users to create their own themes.

**Comment by JasonM77** [comment]
Regarding #5112, it would be good to get some better contrast between the background and foreground colours, especially in the Collection list and the Requests within the Collections. I don't want to use Dark Mode.

![image](https://user-images.githubusercontent.com/9814304/59897501-46485f80-9430-11e9-9cc9-ae5e928234be.png)

**Comment by xeruf** [comment]
See https://github.com/postmanlabs/postman-app-support/issues/3843 and https://github.com/postmanlabs/postman-app-support/issues/2450. Both could easily be solved by adding custom theme support.

**Comment by Wotuu** [comment]
What's the status on this? I'm mostly interested in changing colors of the http methods which are inconsistent with Swagger.

![image](https://user-images.githubusercontent.com/811809/68858682-da6bd980-06e5-11ea-85cd-b89f010ad92f.png)

**Comment by roennow** [comment]
So has this been taking completely off the road map?

**Comment by tristinDLC** [comment]
This is still an open issue, but I see that it's a few years old. Has custom theming been removed development? I have many other apps all themed the same and it would be nice to utilize the same setup in Postmaster since it's a commonly used app of mine.

**Comment by brorbw** [comment]
+1 for custom themes. Having an editor and a terminal with the same themes and postman default themes looks odd. Even slack allows for custom themes. And that is also an electron application

**Comment by bgrabar** [comment]
In v.8, the light theme also provides too little contrast. The beige-ish background in the Collections pane makes it difficult for me to read my list of requests. Increasing the font size doesn't help, because the font gets too grainy. I would prefer the option to have a white background, as v.7 had. I'm over 40, and my eyes aren't what they use to be. This is an accessibility issue for me.

**Comment by pogilvieCB** [comment]
I have a vision diaability and the low contrast of the default themes make postman unpleasant to use.   I would use postman more if I could increase the contrast.

**Comment by erich5168** [comment]
+1 or just an option where you can add a background image and have the abiliity to adjust opacity... ( Seen on Intellij IDE)
![image](https://user-images.githubusercontent.com/31515782/126099868-1b9bb166-11dd-49f2-b204-067590c26929.png)


I have multiple chrome windows... Sometimes when it gets hectic its hard to find POSTMAN.

**Comment by jrcii** [comment]
I want to control every font face, color, size, and background color in the app-- I have to look at it for hours every day for my job. Thank you!!

**Comment by jurfalino** [comment]
+1 dark theme is just not usable, postman would really benefit from the community here. at least being able to share themes as JSON files.

**Comment by j0eii** [comment]
darktheme is just too dark
white theme is just too bright
may i just ask for a small feature update that can allow us to customize the background color?
because it just hurts my eyes...
postman is a great dev/pentest tool all across the years..
need this feature..

**Comment by krakadzeyau-clarabridge** [comment]
Four years passed... Shame on you, Postman team.

**Comment by jrcii** [comment]
A year passed since my comment, still the only tool in my dev tools that I can't control the appearance of. My IDE that is 10x more complex than Postman I can completely theme.

**Comment by ghost** [comment]
At least let us theme syntax, the current colors are horrible.

**Comment by timmyha** [comment]
Postman is incredibly painful to use with the two themes available. Please reconsider this.

**Comment by TKIPisalegacycipher** [comment]
It would be really dope if I could theme Postman.

**Comment by devon-wolf** [comment]
Another vote for themable Postman! I rely on it more and more and have been really happy with the functionality but the colors are just unpleasant to me on both dark and light and that keeps me out of the tool more than I want. (I tend to keep my tools gray/dim rather than dark or light, much easier on my eyes.)

**Comment by ron-arten** [comment]
Adding a +1 for theme editing.

**Comment by PederFosse** [comment]
+1 For editing theme.

**Comment by esanchezrocket** [comment]
I have upvoted this feature request. Dying to be able to get the Dracula Theme for Postman!

**Comment by sam-lucas** [comment]
🙏  theme editing pls

**Comment by ClemensKlapp** [comment]
theeming is necessary!

**Comment by TiagoRibeiro25** [comment]
Community themes support pls 😎✌️

**Comment by mesidex** [comment]
+1, I'd love to be able to modify the default line height

**Comment by risalfajar** [comment]
It is 2024 already, come on Postman you can do better

**Comment by jshroads** [comment]
It really is painful.  Programming with eyes older than dirt is just painful!!

**Comment by ThinGlass** [comment]
6 years later, still no option to customize the theme. I guess we need more votes

**Comment by JulienCoyard** [comment]
+1 For editing theme

**Comment by irisida** [comment]
Urge to consider prioritizing this. In this day and age your tool is being left behind in terms of customisation.  It's been multiple years. Embrace the community.

**Comment by ZeChArtiahSaher** [comment]
Ong just allowssetting something different, this dark theme is utter tra**.

**Comment by steinhjertas** [comment]
+1

**Comment by arthurvanl** [comment]
+1

**Comment by ZeChArtiahSaher** [comment]
How can one drop v11 add custom colors to workspaces and not user settings, tf. Some kind of dystopian backwards product restriction. WE MUST ENFORCE WORKSPACE COLORS, like what's up? It's just a requests client chill. Just give users the ability to have the same workspace settings but for their client as override and it's done deal?

**Comment by bbollig** [comment]
+1

**Comment by chris-torruella** [comment]
Hey everyone, we heard you and I'm excited to share that Postman v11 introduces a whole set of application themes accessible via Settings/Themes. Now you can also sync these with your System preferences. But very importantly, we've also included High Contrast themes for enhanced contrast as needed and Workspace custom theming for differentiation. We hope these will help to elevate your experience, and we're eager to hear your feedback to keep progressing. Cheers!

**Comment by keval-codezee876** [comment]
> Hey everyone, we heard you and I'm excited to share that Postman v11 introduces a whole set of application themes accessible via Settings/Themes. Now you can also sync these with your System preferences. But very importantly, we've also included High Contrast themes for enhanced contrast as needed and Workspace custom theming for differentiation. We hope these will help to elevate your experience, and we're eager to hear your feedback to keep progressing. Cheers!

doesn't address the issue. having prebuilt themes is not enough. open your api to allow custom colors
